docker-compose-separate-containers.yml

```diff
@@ -0,0 +1,34 @@
+version: '2'
+services:
+  nginx:
+    image: nginx
+    container_name: nginx
+    ports:
+      - "80:80"
```

Comment:
I'm not sure I see the point given that there aren't any certs. But I can add it if you like.